rasa/validator.py

```diff
@@ -192,6 +192,23 @@ def verify_utterances_in_stories(self, ignore_warnings: bool = True) -> bool:
 
         return everything_is_alright
 
+    def verify_actions_in_stories_rules(self) -> bool:
+        """Verifies that actions used in stories and rules are present in the domain."""
+        everything_is_alright = True
+
+        for story in self.story_graph.story_steps:
+            for event in story.events:
+                if not isinstance(event, ActionExecuted):
+                    continue
+
+                if not event.action_name.startswith("action_"):
+                    continue
+
+                if event.action_name not in self.domain.action_names_or_texts:
+                    self.domain.raise_action_not_found_exception(event.action_name)
```

Comment:
don't we want to warn instead of raise in this case? This would be in line with the other methods here.

Reply:
Ok, can do that in order to maintain consistency - for some reason I thought I should raise the exception as done in `rasa train` based on the issue thread discussion.